policies/templates/gcp_app_service_versions.yaml

```diff
@@ -28,7 +28,6 @@ spec:
               type: integer
               description: "The maxinum number of simultaneous versions
                Defaults to 2"
-              default: 2
```

Comment:
Why can't we have a default here?

Reply:
Constraint framework doesn't allow it.

Reply:
Got it. Can we open an issue/fix upstream, since it seems like something CF should allow?

Reply:
Filed https://github.com/open-policy-agent/frameworks/issues/70

Reply:
Also, keep in mind that FCV never actually did anything with that field, the change here is that rather than ignoring the default, FCV now errors out.

Reply:
Right, I think we can remove for now until that's fixed.